Specs/Choice/Hindi/BooleanModel.json

```diff
@@ -0,0 +1,213 @@
+[
+  {
+    "Input": "बिलकुल!",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "बिलकुल",
+        "TypeName": "boolean",
+        "Resolution": {
+          "value": true,
+          "score": 1.0
```

Comment:
Scores will very likely need to be recalculated in specs once patterns are translated/fixed and tests re-enabled.